utils/rest_api/create_task.py

```diff
@@ -0,0 +1,259 @@
+import argparse
+from argparse import ArgumentParser
+import getpass
+import requests
+from pprint import pprint
+import json
+import logging
+import http.client as http_client
+import time
+
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning) # to suppress warnings that "Unverified HTTPS request is being made"
+
+def set_http_verbose():
+    http_client.HTTPConnection.debuglevel = 1
+
+    logging.basicConfig()
+    logging.getLogger().setLevel(logging.DEBUG)
+    requests_log = logging.getLogger("requests.packages.urllib3")
+    requests_log.setLevel(logging.DEBUG)
+    requests_log.propagate = True
+
+def print_hrule(name = ""):
+    if name:
+        print("=" * 5 + " " + str(name) + " " + "=" * 80)
+    else:
+        print("=" * 80)
+
+def get_user_password_from_file(password_file):
+    with open(password_file) as f:
+        for line in f:
+            line = line.strip()
+            if ':' not in line or line.startswith('#'):
+                continue
+            chunks = line.split(':')
+            assert len(chunks) == 2, "Wrong password_file {}".format(password_file)
+            return chunks
+    raise RuntimeError("Wrong password file")
+
+def get_user_password_from_input():
+    user = getpass.getuser()
+    answer = input("User = {}? [Y/n]: ".format(user))
+    if answer and not answer.lower().startswith('y'):
+        user = input('User: ')
+
+    password = getpass.getpass()
+    return user, password
+
+def create_task(full_relative_path, bug_tracker, cur_user_info, user, password, labels_descr, cvat_uri):
+    res = {"status": False, "full_relative_path": full_relative_path}
+
+    data_task_create = {
+        "name": full_relative_path,
```

Comment:
All this parameters can be inside `--task-config`. You can provide some placeholder for "full_relative_path" and use it inside `--task-config`.

Reply:
Nikita, I'd prefer not to move all these parameters to one "--task-config".
At least, since most of them should be rewritten by the script -- I mean the fields `bug_tracker`, `owner`, and `name`.

So, I'd prefer to keep `--labels-config` parameter.